### PR TITLE
Unselectable stamp #1752

### PR DESCRIPTION
**Pull request by VSBalanchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less [hunk 1]
```diff
@@ -1,29 +1,28 @@
 .annualreport-modal {
-    .textCenter {
-        text-align: center;
-    }
-    
-    .container {
-        width: 100%;
-    }
+  .textCenter {
```
Comment (OleksandrBilan) [inline src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less:2]: Why do you change tabulation from 4 spaces to 2?

**Comment by SemeniukIhor** [comment]
Я не менял!

Отправлено с iPhone

File: src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less (hunk 1 above)
Comment (VSBalanchuk) [reply src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less:2]: I use Visual Studio Code extension "Prettier - Code formatter", it changed tabulation. Is it bad practise to do so (use 2 spaces for tabulation) and i should rewrite it back to 4 spaces?

Comment (OleksandrBilan) [reply src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less:2]: As i remember, there is no any format rules for tabulation on EPlast, but you don't need to change it in every file. You can leave it with 2 spaces, but it is just some useless work to change the tabulation

**Review by OleksandrBilan — APPROVED** [review]

File: src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less (hunk 1 above)
Comment (VSBalanchuk) [reply src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less:2]: Ok, thanks

**Review by sharix4 — APPROVED** [review]

**Review by DmytroZelinsky — APPROVED** [review]

**Pull request merged**